### PR TITLE
Updated MY list

### PR DESCRIPTION
**Pull request by sneft**

### CODE REVIEW

File: lists/my.csv [hunk 1]
```diff
@@ -293,3 +293,58 @@ http://www.ytl.com.my,VOIP,VOIP,2014-04-15,citizenlab,
...
+https://globalvoicesonline.org/2015/08/10/protests-as-malaysia-suspends-two-newspapers-following-a-corruption-expose/,FEXP,Free expression and media freedom,2016-04-18,citizenlab,
+http://www.sarawakreport.org/,FEXP,Free expression and media freedom,2016-04-18,citizenlab,
+http://www.theedgemarkets.com/my,FEXP,Free expression and media freedom,2016-04-18,citizenlab,
+http://www.wsj.com/articles/SB10130211234592774869404581083700187014570,FEXP,Free expression and media freedom,2016-04-18,citizenlab,
+http://www.bersih.org/,POLR,Political reform,2016-04-18,citizenlab,
+https://medium.com/@sarawakreport,FEXP,Free expression and media freedom,2016-04-18,citizenlab,
+https://medium.com/@sarawakreport/half-million-dollar-denial-najib-s-latest-1mdb-denial-is-directed-against-obama-s-key-aide-e9a46c71a210#.iwyok5h21,FEXP,Free expression and media freedom,2016-04-18,citizenlab,
+https://medium.com/@sarawakreport/najib-negotiates-his-exit-but-he-wants-safe-passage-and-all-the-money-exclusive-disclosure-8d7d44343b50#.rm2d681n3,FEXP,Free expression and media freedom,2016-04-18,citizenlab,
```
Comment (hellais) [inline lists/my.csv]: If the URLs are in HTTPS it doesn't make sense to include the path in the URL. I would suggest having 1 in HTTPS and the others be in HTTP.

File: lists/my.csv [hunk 2]
```diff
@@ -293,3 +293,58 @@ http://www.ytl.com.my,VOIP,VOIP,2014-04-15,citizenlab,
...
+https://vulcanpost.com/353991/bersih-4-0-global/,POLR,Political reform,2016-04-18,citizenlab,
+https://vulcanpost.com/351121/10-items-to-bring-bersih-4-0/,POLR,Political reform,2016-04-18,citizenlab,
+http://www.straitstimes.com/asia/se-asia/what-you-need-to-know-about-malaysias-bersih-movement,POLR,Political reform,2016-04-18,citizenlab,
+http://www.therakyatpost.com/news/2015/08/25/bersih-4-0-gets-green-light-from-sarawak-police/,POLR,Political reform,2016-04-18,citizenlab,
+http://www.malaysiakini.com/news/306575,POLR,Political reform,2016-04-18,citizenlab,
+http://www.malaysiakini.com/news/310156,POLR,Political reform,2016-04-18,citizenlab,
+http://www.tallypress.com/fun/10-important-items-to-bring-to-bersih-4-rally,POLR,Political reform,2016-04-18,citizenlab,
+https://www.article19.org/resources.php/resource/38089/en/malaysia:-blocking-websites-to-prevent-protest-violates-international-law,HUMR,Human rights,2016-04-18,citizenlab,
+https://globalvoicesonline.org/2015/08/10/protests-as-malaysia-suspends-two-newspapers-following-a-corruption-expose/,FEXP,Free expression and media freedom,2016-04-18,citizenlab,
```
Comment (hellais) [inline lists/my.csv]: Same thing here as bellow, I would perhaps include this as a HTTP URL.

File: lists/my.csv [hunk 3]
```diff
@@ -293,3 +293,58 @@ http://www.ytl.com.my,VOIP,VOIP,2014-04-15,citizenlab,
...
+https://www.facebook.com/seacem,FEXP,Free expression and media freedom,2016-04-18,citizenlab,
+http://www.facebook.com/boikotutusan,FEXP,Free expression and media freedom,2016-04-18,citizenlab,
+http://www.facebook.com/pages/barisan-nasional/219708793830,POLR,Political reform,2016-04-18,citizenlab,
+http://youtu.be/edaqtgy-azq,FEXP,Free expression and media freedom,2016-04-18,citizenlab,
+https://www.facebook.com/jkkfc,FEXP,Free expression and media freedom,2016-04-18,citizenlab,
+http://www.youtube.com/watch?&v=ui7xb3d2rna,FEXP,Free expression and media freedom,2016-04-18,citizenlab,
+http://www.youtube.com/watch?v=_1rrnggnm6a,FEXP,Free expression and media freedom,2016-04-18,citizenlab,
+http://pru13.info,POLR,Political reform,2016-04-18,citizenlab,
+https://www.facebook.com/supportprdap,FEXP,Free expression and media freedom,2016-04-18,citizenlab,
+https://www.facebook.com/Bersih4.0Malaysia,POLR,Political reform,2016-04-18,citizenlab,
```
Comment (hellais) [inline lists/my.csv]: These should go in HTTPS

File: lists/my.csv [hunk 4]
```diff
@@ -293,3 +293,58 @@ http://www.ytl.com.my,VOIP,VOIP,2014-04-15,citizenlab,
...
+http://www.themalaymailonline.com/malaysia/article/cops-cannot-declare-may-day-rally-illegal-lawyer-says,FEXP,Free expression and media freedom,2016-04-18,citizenlab,
+http://www.themalaymailonline.com/malaysia/article/on-may-day-trade-unions-to-rally-against-gst-tppa,FEXP,Free expression and media freedom,2016-04-18,citizenlab,
+http://www.malaysiakini.com/news/261006,FEXP,Free expression and media freedom,2016-04-18,citizenlab,
+http://www.malaysiakini.com/news/228819,FEXP,Free expression and media freedom,2016-04-18,citizenlab,
+https://www.facebook.com/photo.php?v=10151567065649375,MMED,Multimedia sharing,2016-04-18,citizenlab,
+https://www.facebook.com/seacem,FEXP,Free expression and media freedom,2016-04-18,citizenlab,
+http://www.facebook.com/boikotutusan,FEXP,Free expression and media freedom,2016-04-18,citizenlab,
+http://www.facebook.com/pages/barisan-nasional/219708793830,POLR,Political reform,2016-04-18,citizenlab,
+http://youtu.be/edaqtgy-azq,FEXP,Free expression and media freedom,2016-04-18,citizenlab,
+https://www.facebook.com/jkkfc,FEXP,Free expression and media freedom,2016-04-18,citizenlab,
```
Comment (hellais) [inline lists/my.csv]: This should be HTTP.

File: lists/my.csv [hunk 5]
```diff
@@ -293,3 +293,58 @@ http://www.ytl.com.my,VOIP,VOIP,2014-04-15,citizenlab,
...
+http://www.themalaymailonline.com/malaysia/article/may-day-rally-organisers-to-disregard-police-ban,FEXP,Free expression and media freedom,2016-04-18,citizenlab,
+http://asalkanbukanumno.webs.com,POLR,Political reform,2016-04-18,citizenlab,
+http://www.pakatanrakyat.my,POLR,Political reform,2016-04-18,citizenlab,
+http://www.themalaymailonline.com/malaysia/article/cops-cannot-declare-may-day-rally-illegal-lawyer-says,FEXP,Free expression and media freedom,2016-04-18,citizenlab,
+http://www.themalaymailonline.com/malaysia/article/on-may-day-trade-unions-to-rally-against-gst-tppa,FEXP,Free expression and media freedom,2016-04-18,citizenlab,
+http://www.malaysiakini.com/news/261006,FEXP,Free expression and media freedom,2016-04-18,citizenlab,
+http://www.malaysiakini.com/news/228819,FEXP,Free expression and media freedom,2016-04-18,citizenlab,
+https://www.facebook.com/photo.php?v=10151567065649375,MMED,Multimedia sharing,2016-04-18,citizenlab,
+https://www.facebook.com/seacem,FEXP,Free expression and media freedom,2016-04-18,citizenlab,
```
Comment (hellais) [inline lists/my.csv]: Same as bellow.

File: lists/my.csv [hunk 6]
```diff
@@ -293,3 +293,58 @@ http://www.ytl.com.my,VOIP,VOIP,2014-04-15,citizenlab,
...
+http://youtu.be/edaqtgy-azq,FEXP,Free expression and media freedom,2016-04-18,citizenlab,
+https://www.facebook.com/jkkfc,FEXP,Free expression and media freedom,2016-04-18,citizenlab,
+http://www.youtube.com/watch?&v=ui7xb3d2rna,FEXP,Free expression and media freedom,2016-04-18,citizenlab,
+http://www.youtube.com/watch?v=_1rrnggnm6a,FEXP,Free expression and media freedom,2016-04-18,citizenlab,
+http://pru13.info,POLR,Political reform,2016-04-18,citizenlab,
+https://www.facebook.com/supportprdap,FEXP,Free expression and media freedom,2016-04-18,citizenlab,
+https://www.facebook.com/Bersih4.0Malaysia,POLR,Political reform,2016-04-18,citizenlab,
+https://en.wikipedia.org/wiki/Bersih_4_rally,POLR,Political reform,2016-04-18,citizenlab,
+https://vulcanpost.com/353991/bersih-4-0-global/,POLR,Political reform,2016-04-18,citizenlab,
+https://vulcanpost.com/351121/10-items-to-bring-bersih-4-0/,POLR,Political reform,2016-04-18,citizenlab,
```
Comment (hellais) [inline lists/my.csv]: Convert to HTTP.

**Comment by sneft** [comment]
Hey Arturo,

This is a debate we've had internally a number of times. It's obvious inefficient to repeatedly test the full path of HTTPS URLs for the same domain, but at the same time we didn't want to make any initial assumptions in constructing the lists that would limit their usefulness in the long-run. In other words, trying to future-proof the lists, leave open the possibility that someone's being MITM'd, etc.

We generally have settled on being exhaustive (in some cases testing both HTTP/HTTPS versions) as the performance impact wasn't our primary consideration.

One middle ground may be to include a small sample of HTTPS full-path URLs (e.g. a handful of full HTTPS URLs of sensitive Twitter accounts or YouTube videos) while testing the HTTP version of the rest. 

Thoughts?

**Comment by hellais** [comment]
@sneft I do see your points.

One thing to take into consideration is the fact that if you only do testing for the HTTPS version of the site you will not catch cases in which the specific path is being filtered by a transparent proxy that doesn't do MITM.

I guess in the end it's probably up to the tool that does the measurements to take into account the fact that it should be testing the URL in both it's HTTPS and HTTP version.

That said if we decide to include in the URLs to test the HTTPS version of websites it should be consistent across all of them. That is if the site does support HTTPS the URL should always be listed using https. It will then be up to the tool consuming the testing list to take into account the fact it should be testing for the URL in both it's HTTPS and HTTP version.

**Comment by hellais** [comment]
Since there are many inconsistencies of this sort across this repository I would say we proceed in merging this and defer the fixing the HTTPS vs HTTP issue by discussing it in a ticket and applying it across all the repo.

**Comment by hellais** [comment]
@sneft could you re-push this branch (for some reason it seems to have disappeared and github lists it as `unknown repository`) ideally rebasing it against master, though I can also do that as part of merging it.

**Comment by sneft** [comment]
I'll re-push these changes separately.

**Pull request closed without merging**